src/libraries/Microsoft.CSharp/src/ILLink/ILLink.Suppressions.xml

```diff
@@ -139,5 +139,29 @@
       <property name="Scope">member</property>
       <property name="Target">M:Microsoft.CSharp.RuntimeBinder.SymbolTable.AddNamesInInheritanceHierarchy(System.String,System.Collections.Generic.List{System.Type})</property>
     </attribute>
+    <attribute fullname="System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessageAttribute">
```

Comment:
> which should not be affected by this PR

This isn't true. `Microsoft.CSharp` references System.Linq.Expressions:

https://github.com/dotnet/runtime/blob/118c0fd093dad025bb8ae37e85a77440e33dbb51/src/libraries/Microsoft.CSharp/src/Microsoft.CSharp.csproj#L245

Since I'm adding annotations to System.Linq.Expressions (specifically RequiresUnreferencedCode attributes), it is causing more warnings in the downstream assembly. For example, this is now unsafe:

https://github.com/dotnet/runtime/blob/1d9e50cb4735df46d3de0cee5791e97295eaf588/src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/ComInterop/CurrencyArgBuilder.cs#L24-L27

Reply:
Thanks - didn't realize that there are dependencies...